service/awsconfig/v10/resource/cloudformation/current.go

```diff
@@ -125,6 +125,8 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 			MasterInstanceType:         masterInstanceType,
 			MasterCloudConfigVersion:   masterCloudConfigVersion,
 
+			Status: stackStatus,
```

Comment:
We set the current stack status to our stack state in order to decide if we should proceed with the deletion or fail for retries.